File: composeApp/src/androidMain/kotlin/co/jarias/flexapp/ui/screens/bingo/card_scanner/BingoCardScannerScreenState.kt
package co.jarias.flexapp.ui.screens.bingo.card_scanner

data class BingoCardScannerScreenState(
    val gameId: Long = 0,
    val isLoading: Boolean = false,
    val isProcessing: Boolean = false,
    val detectedGrid: List<List<String>>? = null,
    val errorMessage: String? = null,
    val scanErrorMessage: String? = null,
    val cardSaved: Boolean = false,
    val navigateBack: Boolean = false
)

File: iosApp/iosApp/Screens/bingo/card_scanner/BingoCardScannerScreenState.swift
import Foundation

struct BingoCardScannerScreenState {
    var gameId: Int64 = 0
    var isLoading: Bool = false
    var isProcessing: Bool = false
    var detectedGrid: [[String]]? = nil
    var errorMessage: String? = nil
    var scanErrorMessage: String? = nil
    var cardSaved: Bool = false
    var navigateBack: Bool = false
}
